### ISSUE TITLE
Battle Opening missing background 

### ISSUE THREAD

**Issue by AaronStardust08** [shaddatic/sa2b-render-fix#67]
I have this mod paired with the Cutscene Revamp mod and for some reason, the background in the Battle opening is all black until the end.  Is this a common problem?  And is there a solution?

**Comment by shaddatic** [comment]
The background is facing the wrong way, this is the case in the vanilla cutscene too. As long as your Render Fix is up to date, there's a config option to disable backface culling in cutscenes as a temporary measure, it is on by default

**Comment by AaronStardust08** [comment]
Will disabling it hurt other cutscenes?

**Comment by shaddatic** [comment]
Yes, many other cutscenes have the missing skybox issue, or worse, so it's recommended you leave it on for now

**Comment by AaronStardust08** [comment]
You say it’s a temporary solution.  Does that mean a better one is being looked at?

**Comment by shaddatic** [comment]
All the cutscenes will be replaced at some point in the future, taking full advantage of backface culling and other features of Render Fix. Any vanilla issues like this will be fixed then, and the workaround will either be default-off or just removed.

Until then, closing issue
